object-store/src/main/java/org/hypertrace/config/objectstore/IdentifiedObjectStoreWithFilter.java

```diff
@@ -23,6 +23,13 @@ protected IdentifiedObjectStoreWithFilter(
     super(configServiceBlockingStub, resourceNamespace, resourceName, configChangeEventGenerator);
   }
 
+  protected IdentifiedObjectStoreWithFilter(
```

Comment:
Hard pressed to think of a reason where we want to support this constructor - it was left on the super class for backwards compatibility, but we want to send config change events for newly created stores always.